pkg/manager/member/pd_failover.go

```diff
@@ -66,7 +66,7 @@ func (f *pdFailover) Failover(tc *v1alpha1.TidbCluster) error {
 	}
 
 	pdDeletedFailureReplicas := tc.GetPDDeletedFailureReplicas()
-	if pdDeletedFailureReplicas >= *tc.Spec.PD.MaxFailoverCount {
+	if pdDeletedFailureReplicas >= *tc.Spec.PD.MaxFailoverCount && pdDeletedFailureReplicas > 0 {
```

Comment:
For your case, you have set `Spec.PD.MaxFailoverCount=0`, right? Which means that you want to disable failover for PD, in this case, I think we should update the code here https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/pd_member_manager.go#L241 as below:
`if m.deps.CLIConfig.AutoFailover && tc.Spec.PD.MaxFailoverCount != nil && *tc.Spec.PD.MaxFailoverCount > 0 {`
WDYT?

Reply:
@DanielZhangQD Thanks for your reply. Yes, we think that your suggestion is correct, and we have updated the code in the latest commit in this PR now.

Reply:
What happens if failover occurs and set the max failover count to 0?